.github/workflows/enforce-toolchain-synchronization.yml

```diff
@@ -17,9 +17,9 @@ on:
   push:
     branches:
       - '**'  # Triggers on pushes to any branch
-
 jobs:
   check-prs:
+    if: ${{ github.event.before != '0000000000000000000000000000000000000000' }}  # Skip for new branches
```

Comment:
Looks cryptic - is this the only option? ${{ !github.event.created }} ? docs: "created, boolean
whether this push created the ref."